### PR TITLE
NXDRIVE-571: Remove use of "triggerRemoteJob" for "build"

### PR DESCRIPTION
**Pull request by BoboTiG**
Much better now. If the triggered job fails, the current job will too.

### CODE REVIEW

File: tools/release.sh [hunk 1]
```diff
@@ -65,41 +67,67 @@ create_beta() {
     echo ">>> [beta ${drive_version}] Creating the commit"
     change_version "${drive_version}"
     git commit -m "Release $drive_version"
-    git push origin master
+    if [ "${DRY_RUN}" = true ]; then
+        echo "DRY-RUN git push origin master"
+        git log -n 5 --abbrev-commit --pretty=oneline
```
Comment (ataillefer) [inline tools/release.sh]: Could use the `-n, --dry-run` option for `git push`

File: tools/release.sh [hunk 2]
```diff
@@ -65,41 +67,67 @@ create_beta() {
...
     changelog "${drive_version}"
 
     echo ">>> [beta ${drive_version}] Creating the tag"
-    git tag "release-${drive_version}"
-    git push origin --tags
+    if [ "${DRY_RUN}" = true ]; then
+        echo "DRY-RUN git tag \"release-${drive_version}\""
+        echo "DRY-RUN git push origin --tags"
+    else
+        git tag "release-${drive_version}"
```
Comment (ataillefer) [inline tools/release.sh]: You can tag even in dry-run as long as you don't push the tag

File: tools/release.sh [hunk 3]
```diff
@@ -65,41 +67,67 @@ create_beta() {
...
+        git push origin --tags
```
Comment (ataillefer) [inline tools/release.sh]: If DRY_RUN: `git push origin --tags --dry-run`

File: tools/release.sh (hunk 2 above)
Comment (BoboTiG) [inline tools/release.sh]: Of course.

File: tools/release.sh (hunk 1 above)
Comment (BoboTiG) [inline tools/release.sh]: Done.

**Pull request merged**